internal/db/schema/migrations/oss/postgres/80/06_auth_token_base_table_updates.up.sql

```diff
@@ -3,6 +3,53 @@
 
 begin;
 
+  -- Add the scope ID column from the account so we can
+  -- use it in the list queries.
+  -- Note, we cannot make this wt_scope_id as it would require the user
+  -- to provide the scope ID at insert time instead of assigning it
+  -- from the account in a trigger. The problem is that domain type constraints
+  -- run before insert triggers, and the wt_scope_id domain type requires
+  -- the scope ID to be at least 10 characters long, so we can't leave it unset on insert.
+  alter table auth_token add column scope_id text;
```

Comment:
Adding `scope_id` to `auth_token` denormalizes the table and nothing in this PR guards against the data anomalies that result from the denormalization. Why not join `auth_token` with `auth_account` in the queries used for listing auth tokens instead?

Reply:
There is an insert trigger that assigns the scope_id from the auth token (so the user can't set something that isn't the right value), and we make the scope_id immutable after the insert (so the user cannot update it after it's been set), is that not sufficient?

I'm happy to change it to a join in the query but we've been worried that doing a join in the initial CTE would make it less performant. I'll make the update here and we can come back to this option if we find the other one is not performant enough.

Reply:
I've changed this to join in the query instead